HTTPShortcuts/curl_command/src/main/kotlin/ch/rmy/curlcommand/CurlCommand.kt

```diff
@@ -122,6 +126,16 @@ class CurlCommand internal constructor() : Serializable {
             curlCommand.silent = true
         }
 
+        fun ipVersion4() = also {
+            curlCommand.ipVersion4 = true
+            curlCommand.ipVersion6 = false
+        }
+
+        fun ipVersion6() = also {
+            curlCommand.ipVersion6 = true
+            curlCommand.ipVersion4 = false
+        }
```

Comment:
I'm following the convention stated at the [curl manpage](https://curl.se/docs/manpage.html):
> Providing --ipv6 multiple times has no extra effect.
> This option is mutually exclusive with -4, --ipv4

You can invoke `curl` with both the `-4` and `-6` flags. If done so, only the last one will be considered. This is also true of this implementation.